Jenkinsfile

```diff
@@ -66,24 +74,6 @@ pipeline {
                 '''
             }
         }
-
-        stage('Lint Integration Tests') {
-            steps {
-                sh './bin/lint_tests'
-            }
-        }
-
-        stage('Lint Spec File') {
-            steps {
-                sh './bin/lint_spec'
```

Comment:
Runs in https://github.com/cyberark/conjur-openapi-spec/blob/main/.github/workflows/linting.yml